### PR TITLE
Stabilize biome-key-management feature

### PR DESCRIPTION
**Pull request by RyanLassigBanks**
Signed-off-by: Ryan Banks <rbanks@bitwise.io>

### CODE REVIEW

File: libsplinter/src/biome/rest_api/mod.rs [hunk 1]
```diff
@@ -117,7 +117,7 @@ use crate::rest_api::sessions::AccessTokenIssuer;
 /// * `GET /biome/user/{id}` - Retrieve user with specified ID
 /// * `DELETE /biome/user/{id}` - Remove user with specified ID
 pub struct BiomeRestResourceManager {
-    #[cfg(any(feature = "biome-key-management", feature = "biome-credentials",))]
+    #[cfg(feature = "biome-credentials")]
```
Comment (vaporos) [inline libsplinter/src/biome/rest_api/mod.rs:120]: These guards would presumably still be necessary when we have the feature turned off?

**Comment by vaporos** [comment]
In the other store modules, key_management::store::diesel::postgres::DieselKeyStore is up a level (we don't have a postgres-specific module in that location. Probably that should be moved.

File: libsplinter/src/biome/rest_api/mod.rs (hunk 1 above)
Comment (RyanLassigBanks) [reply libsplinter/src/biome/rest_api/mod.rs:122]: Based on the way the routes are set up, the user store only appears to be required by the routes that are enabled by the `biome-credentials` feature

Comment (vaporos) [reply libsplinter/src/biome/rest_api/mod.rs:122]: Ok, please move that fix to another PR.

**Pull request merged**